packages/eui/src/components/steps/step.tsx

```diff
@@ -42,9 +42,9 @@ export interface EuiStepInterface {
    */
   status?: EuiStepStatus;
   /**
-   * Title sizing equivalent to EuiTitle, but only `m`, `s` and `xs`. Defaults to `s`
+   * Title sizing equivalent to EuiTitle, but only `m`, `s`, `xs` and `xxs`. Defaults to `s`
    */
-  titleSize?: Exclude<EuiTitleProps['size'], 'xxxs' | 'xxs' | 'l'>;
+  titleSize?: Exclude<EuiTitleProps['size'], 'xxxs' | 'l'>;
```

Comment:
Ooh I think I get the discrepancy now. EuiSteps uses `titleSize` and EuiStepsHorizontal just uses `size`.

I think we need to clarify the wording on this to mention that `xxs` doesn't actually reduce the **title** font size, only the corresponding icon.

```suggestion
   * Title sizing equivalent to **EuiTitle**, but only `m`, `s`, `xs` font sizes.
   * The `xxs` size reduces the size of the accompanying step indicator, but not the title itself.
   * @default s
   */
  titleSize?: Pick<EuiTitleProps['size'], 'xxs' | 'xs' | 's' | 'm'>;
```

Also tweaking the `Exclude` as I'm not a huge fan of it, `Pick<>` feels more intuitive to me

Reply:
Updated [here](https://github.com/elastic/eui/pull/7813/commits/712476c576fa247bec23235cdc0acfc96a5c0732#diff-223e29eb0f3196aeaf43902a9b4a3fc1c71af3ceaa8e93a44b59f75763b0ca2aR59).